src/App.js

```diff
@@ -2,78 +2,133 @@
+        setTimeout(() => {
+          setTimer(gameTime);
+        }, 1000);
+      }
+    }
+    return () => {
+      clearInterval(interval);
+    };
+  }, [timeOn, time]);
 
   return (
-    <div className="container">
-      <section className="scoreboard">
-        <div className="topRow">
-          <div className="home">
-            <h2 className="home__name">{homeTeam}</h2>
+    <div className='container'>
+      <section className='scoreboard'>
+        <h2 className='time_name'>Time</h2>
+        <div className='topRow'>
+          <div className='home'>
+            <h2 className='home__name'>{homeTeam}</h2>
 
             {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
 
-            <div className="home__score">{homeScore}</div>
+            <div className='home__score'>{homeScore}</div>
+          </div>
+
+          <div className='time'>
+            <div className='timer'>{time}</div>
+            {/* <div className='timer'>{time}</div>
+            <div className='timer'>{time}</div>
+            <div className='timer'>{time}</div> */}
           </div>
-          <div className="timer">{time}</div>
-          <div className="away">
-            <h2 className="away__name">{awayTeam}</h2>
-            <div className="away__score">{awayScore}</div>
+
+          <div className='away'>
+            <h2 className='away__name'>{awayTeam}</h2>
+            <div className='away__score'>{awayScore}</div>
           </div>
         </div>
-        <BottomRow />
+        <BottomRow quarter={quarter} />
       </section>
-      <section className="buttons">
-        <div className="homeButtons">
+      <section className='buttons'>
+        <div className='homeButtons'>
           {/* TODO STEP 4 - Now we need to attach our state setter functions to click listeners. */}
           {/* - A touchdown is worth 7 points (assume the following extra point is made) - A field goal is worth 3 points */}
 
           {/* <HomeTouchBtn homeScore={}/> */}
 
           <button
-            className="homeButtons__touchdown"
-            onClick={event => setHomeScore(homeScore + 7)}
-          >
+            className='homeButtons__touchdown'
+            onClick={event => setHomeScore(homeScore + 7)}>
             Home Touchdown
           </button>
 
           <button
-            className="homeButtons__fieldGoal"
-            onClick={event => setHomeScore(homeScore + 3)}
-          >
+            className='homeButtons__fieldGoal'
+            onClick={event => setHomeScore(homeScore + 3)}>
             Home Field Goal
           </button>
         </div>
-        <div className="awayButtons">
+        <div className='awayButtons'>
           <button
-            className="awayButtons__touchdown"
-            onClick={event => setAwayScore(awayScore + 7)}
-          >
+            className='awayButtons__touchdown'
+            onClick={event => setAwayScore(awayScore + 7)}>
```

Comment:
I would avoid defining variables that aren't used. Studies show that people can only store 7 things in their head at any given time so defining `event` even though you don't use it in this function is what I would consider to be an Anti Pattern and I would recommend not having any arguments instead OR using a `_` before event like `_event` to indicate it is not used.